src/containers/shared/components/Transaction/NFTokenMint/types.ts

```diff
@@ -0,0 +1,14 @@
+import { TransactionCommonFields } from '../types'
+
+export interface NFTokenMint extends TransactionCommonFields {
+  NFTokenTaxon: number
+  Issuer?: string
+  TransferFee?: number
+  URI?: string
```

Comment:
I think `Flags` is an important field to display potentially. Not sure if they were included in the original design of the page - but they do dramatically change what the end result NFTOken does.

Reply:
They can.  This will need to be a redesign.  Right now flags are relegated to the Details tab.

Reply:
Sounds good - won't let it affect this PR then.